zigpy/config/__init__.py

```diff
@@ -152,6 +155,38 @@
         vol.Optional(
             CONF_NWK_TC_LINK_KEY, default=CONF_NWK_TC_LINK_KEY_DEFAULT
         ): cv_key,
+        # For background, TX power is used by some end devices to pick parent routers.
+        # If your coordinator is transmitting at +20dBm, end devices will try to pick
+        # the coordinator as their parent, thinking it is close, but it will actually be
+        # too far to hear them. Some recover from this but you may run into end devices
+        # picking the coordinator as a parent when they should have instead picked
+        # a closer router instead.
+        #
+        # Similarly, a higher TX power will cause the coordinator's messages to travel
+        # farther, which can cause it to be heard by more devices, but also "interrupt"
+        # other devices that are transmitting far away, resulting in more retries and
+        # thus less reliability.
+        #
+        # Finally, a Zigbee network requires reliable *bidirectional* communication for
+        # many operations: just because a device is able to hear the coordinator does
+        # not mean that the device can actually send a reply back.
+        #
+        # Tweak this setting at your own risk. Turning it up to 20 is
```

Comment:
nit: What is "turning it up to 20"? 😄 
(-> some negative word(s) missing at the end of that line)